src/models.rs

```diff
@@ -160,6 +160,22 @@ fn default_root_path_value() -> String {
     "/".to_string()
 }
 
+#[derive(Serialize, Deserialize, Clone, Eq, PartialEq, Hash)]
+pub enum Protocol {
+    HTTP,
+}
+
+#[derive(Serialize, Deserialize, Clone, Eq, PartialEq, Hash)]
+pub struct Port {
+    pub id: String,
+    pub long_id: uuid::Uuid,
+    pub port: u16,
```

Comment:
maybe try to find another name, because it feels weird to have Port.port, maybe `number`?

Reply:
done